packages/dd-trace/src/plugins/util/test.js

```diff
@@ -281,16 +281,34 @@ const POSSIBLE_CODEOWNERS_LOCATIONS = [
   '.gitlab/CODEOWNERS'
 ]
 
-function getCodeOwnersFileEntries (rootDir = process.cwd()) {
-  let codeOwnersContent
-
-  POSSIBLE_CODEOWNERS_LOCATIONS.forEach(location => {
+function readCodeOwners (rootDir) {
+  for (const location of POSSIBLE_CODEOWNERS_LOCATIONS) {
     try {
-      codeOwnersContent = fs.readFileSync(`${rootDir}/${location}`).toString()
+      return fs.readFileSync(path.join(rootDir, location)).toString()
     } catch (e) {
       // retry with next path
     }
-  })
+  }
+  return ''
+}
+
+function getCodeOwnersFileEntries (rootDir) {
+  let codeOwnersContent
+  let usedRootDir = rootDir
+  let isTriedCwd = false
+
+  if (!usedRootDir) {
```

Comment:
maybe also compare if `usedRootDir === process.cwd()` ?

```suggestion
  let processCwd = process.cwd();
  
  if (!usedRootDir || usedRootDir == processCwd) {
     usedRootDir = processCwd;
```

Reply:
good idea 👍